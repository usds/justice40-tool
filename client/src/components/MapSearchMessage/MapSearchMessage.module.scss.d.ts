declare namespace MapSearchMessageModuleScssNamespace {
    export interface IMapSearchMessageModuleScss {
      showMessage: string;
      hideMessage: string;
    }
  }

declare const MapSearchMessageModuleScssModule: MapSearchMessageModuleScssNamespace.IMapSearchMessageModuleScss & {
    /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
    locals: MapSearchMessageModuleScssNamespace.IMapSearchMessageModuleScss;
  };

  export = MapSearchMessageModuleScssModule;

