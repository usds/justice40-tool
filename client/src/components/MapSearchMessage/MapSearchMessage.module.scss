@use '../../styles/design-system.scss' as *;

@mixin searchMessageLayout {
    color: red;
    background-color: white;
    @include u-margin-bottom(.5);
    @include u-margin-top(.5);
    @include u-padding-left(1);
}

.showMessage {
    @include searchMessageLayout;
    display: block;
    width: fit-content;
}

.hideMessage {
    @include searchMessageLayout;
    visibility: hidden;
}