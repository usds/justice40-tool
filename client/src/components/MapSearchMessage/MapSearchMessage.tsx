import React from 'react';
import {useIntl} from 'gatsby-plugin-intl';

import * as EXPLORE_COPY from '../../data/copy/explore';
import * as styles from './MapSearchMessage.module.scss';

interface ISearchMessage {
    isSearchResultsNull: boolean;
};

const MapSearchMessage = ({isSearchResultsNull}:ISearchMessage) => {
  const intl = useIntl();

  return (
    <div className={isSearchResultsNull ? styles.showMessage : styles.hideMessage}>
      {intl.formatMessage(EXPLORE_COPY.MAP.SEARCH_RESULTS_EMPTY_MESSAGE)}
    </div>
  );
};

export default MapSearchMessage;
