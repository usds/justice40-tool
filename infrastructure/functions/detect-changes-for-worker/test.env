REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1