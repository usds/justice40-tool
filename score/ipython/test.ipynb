{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
