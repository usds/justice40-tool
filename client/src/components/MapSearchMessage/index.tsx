import MapSearchMessage from './MapSearchMessage';

export default MapSearchMessage;
