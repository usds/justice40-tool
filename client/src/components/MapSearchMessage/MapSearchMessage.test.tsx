import * as React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../test/testHelpers';
import MapSearchMessage from './MapSearchMessage';

describe('rendering of the MapSearchMessage when search results are empty', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <MapSearchMessage isSearchEmpty={true}/>
      </LocalizedComponent>,
  );

  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});

describe('rendering of the MapSearchMessage when search results are not empty', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <MapSearchMessage isSearchEmpty={false}/>
      </LocalizedComponent>,
  );

  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
