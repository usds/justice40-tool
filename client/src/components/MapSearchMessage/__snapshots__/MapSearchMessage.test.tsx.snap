// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of the MapSearchMessage when search results are empty checks if component renders 1`] = `
<DocumentFragment>
  <div>
    Location not found or unknown. Please try a different search.
  </div>
</DocumentFragment>
`;

exports[`rendering of the MapSearchMessage when search results are not empty checks if component renders 1`] = `<DocumentFragment />`;
